src/3d/castletransform.pas

```diff
@@ -447,6 +452,10 @@       TEnumerator = class
     procedure RemoveFromWorld(const Value: TCastleAbstractRootTransform);
     procedure RemoveBehaviorIndex(const BehaviorIndex: Integer);
     procedure SetListenPressRelease(const Value: Boolean);
+
+    { Check current item exist taking into account the existence of the
+      unique parents of this item }
+    function GetRealExistance: Boolean;
```

Comment:
Proper English is "existence" not "existance" as far as I see.

Although "existance" seems to be a common misspelling, some dictionaries redirect it to "existence" automatically ( https://duckduckgo.com/?t=ffab&q=Existance&ia=web and note that it's not DuckDuckGo fixing the misspelling, but various particular pages ).